VideoLocker/src/test/java/org/edx/mobile/profiles/UserProfileAccomplishmentsFragmentTest.java

```diff
@@ -0,0 +1,86 @@
+package org.edx.mobile.profiles;
```

Comment:
For the `UserProfileAccomplishmentsFragmentTest`, are we only testing that the loading indicator works and the page is drawn?